src/views/Wallet/WalletView.vue

```diff
@@ -3,10 +3,9 @@
     <div class="box">
       <div class="columns is-vcentered">
         <div class="column">
-          <p
-            class="is-size-3 is-family-handwritten is-capitalized has-text-grey has-line-height-small"
-            v-html="wallet.name"
-          />
+          <p class="is-size-3 is-family-handwritten is-capitalized has-text-grey has-line-height-small">
+            {{ wallet.name }}
+          </p>
```

Comment:
I would say if we already validated the wallet name do we need to force it into a string?
I mean its not a big deal

Reply:
Do we really want to validate wallet names, where ever they can come from. 

So you would need to clean up wallet names:

* Keychain
* Imports
* User imports 

and maybe more coming. I think the smaller and more valid way is just to fix the HTML rendering instead.
Let the user have his HTML string if he likes to. Maybe people want to call it `<VERGE>`

Reply:
Cool 👍